libraries/AP_HAL_PX4/RCInput.cpp

```diff
@@ -38,6 +40,12 @@ bool PX4RCInput::new_input()
     _last_read = _rcin.timestamp_last_signal;
     _override_valid = false;
     pthread_mutex_unlock(&rcin_mutex);
+    if ((_rcin.input_source != last_input_source) ||
+        (!_rcin.rc_failsafe && last_rc_failsafe)) {
```

Comment:
I don't think we should rebroadcast when we come out of failsafe, it will get annoyingly spammy in a poor radio environment/on the edge of the RC system's range, and the assumption should always be that the radio came back with the same kind of input as before, and the previous line will broadcast it for us if that ever isn't true.